migrations/Version20251212143025.php

```
@@ -0,0 +1,36 @@
+    public function getDescription(): string
+    {
+        return 'Removes legacy page';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->addSql('DELETE FROM caf_page WHERE code_page = \'email-change\'');
+        $this->addSql('DELETE FROM caf_page WHERE code_page = \'user-confirm\'');
+    }
+
+    public function down(Schema $schema): void
```

PHPMD (2.15.0)

Avoid unused parameters such as '$schema'. (undefined)

(UnusedFormalParameter)